### ISSUE TITLE
Include different frameworks compatibility for the model

### ISSUE THREAD

**Issue by harshit98** [harshit98/single-particle-recognition#1]
Implement the single-particle recognition model in the following frameworks:

- [ ] Tensorflow
- [ ] Keras
- [ ] PyTorch

Extras: 
- [ ] Add perfomance comparison table.
- [ ] Plot accuracy vs loss (2 graphs doing comparison).
